pyispyb/app/routes/options.py

```diff
@@ -0,0 +1,40 @@
+from fastapi import Depends, Request
+
+from ...dependencies import permission
+from ..base import AuthenticatedAPIRouter
+from ..extensions.options import base as crud
+from ..extensions.options.schema import Options, UIOptions
+
+router = AuthenticatedAPIRouter(prefix="/options", tags=["Options"])
+
+
+@router.get(
+    "/ui",
+    response_model=UIOptions,
+)
+def get_ui_options() -> UIOptions:
```

Comment:
router is an `AuthenticatedAPIRouter`. Should it not be available without authentication so that the UI can read the options before the user is authenticated?

Reply:
Best to not leak any information for free

Reply:
Good point, is it just auth information you need for the UI before login? If so we could provide this separately in the auth refactor (https://github.com/ispyb/py-ispyb/pull/201) on a specific unauthenticated route?

Reply:
Here is what the UI configuration currently looks like: https://github.com/ispyb/py-ispyb-ui/blob/main/src/config/sites.ts
What we need before login is `authentication`, `name` and `description`. I think `icon` could be removed and the rest served only after login.
The file actually is a list of config for each site, and the user choses which one they want to login to. If each site stores its configuration in the DB, then we can replace it by a simple list of server URLs.

Reply:
Suggest we provide the auth information in #201 and site specific configuration as a new feature on top of this pr